tests/unit/lbrynet_daemon/_sample_file_list.py

```diff
@@ -0,0 +1,237 @@
+if __name__ == '__main__':
+    from sys import exit
+    exit()
+
```

Comment:
Can you remove above conditional and also move this entire `FAKED_ATTRIBUTES` list into the file that imports it, just all the way at the bottom.

Reply:
Boom